Application/EdFi.Ods.Admin.Api/Infrastructure/WebApplicationBuilderExtensions.cs

```diff
@@ -27,24 +26,14 @@ public static class WebApplicationBuilderExtensions
     public static void AddServices(this WebApplicationBuilder webApplicationBuilder)
     {
         var executingAssembly = Assembly.GetExecutingAssembly();
-        webApplicationBuilder.Services.AddAutoMapper(executingAssembly, typeof(AdminManagementMappingProfile).Assembly);
+        webApplicationBuilder.Services.AddAutoMapper(executingAssembly, typeof(AdminApiMappingProfile).Assembly);
         webApplicationBuilder.Services.AddScoped<InstanceContext>();
 
         foreach (var type in typeof(IMarkerForEdFiOdsAdminAppManagement).Assembly.GetTypes())
         {
             if (type.IsClass && !type.IsAbstract && (type.IsPublic || type.IsNestedPublic))
             {
                 var concreteClass = type;
-
-                if (concreteClass == typeof(OdsApiFacade))
-                    continue; //IOdsApiFacade is never resolved. Instead, classes inject IOdsApiFacadeFactory.
-
-                if (concreteClass == typeof(OdsRestClient))
-                    continue; //IOdsRestClient is never resolved. Instead, classes inject IOdsRestClientFactory.
-
-                if (concreteClass == typeof(TokenRetriever))
-                    continue; //ITokenRetriever is never resolved. Instead, other dependencies construct TokenRetriever directly.
```

Comment:
Someday we may need to restore an ODS/API request client. When that time comes, it might be interesting to use the SDK codegen utility to create the required classes (though that will create many more than we actually need in this application; the extra classes can be deleted in that case).